selfdrive/car/chrysler/interface.py

```diff
@@ -24,7 +24,6 @@ def _get_params(ret, candidate, fingerprint, car_fw, experimental_long, docs):
     elif candidate in RAM_DT:
       ret.safetyConfigs[0].safetyParam |= Panda.FLAG_CHRYSLER_RAM_DT
 
-    ret.minSteerSpeed = 3.8  # m/s
```

Comment:
when is this set?

Reply:
https://github.com/commaai/openpilot/blob/master/selfdrive/car/interfaces.py#L123